### DISCUSSION TITLE
why top_region_index and bottom_region_index are variable in MAISI?

### DISCUSSION THREAD

**Discussion by shengzhang90** [Project-MONAI/tutorials#1914] [General] [answered]
Hi, 

In the file of C4KC-KiTS_subset.json, I find that the values of top_region_index and bottom_region_index are variable rather than fixed, e.g., top_region_index might be [0,1,0,0] or [0,0,1,0], bottom_region_index might be [0,0,0,1] or [0,0,1,0] why?

Thanks a lot.

**Comment by Can-Zhao** [comment]
This is to provide condition for the output image

**Comment by Can-Zhao** [comment, marked as answer]
We just released a new MAISI version which will not require top_region_index and bottom_region_index as inputs
